development/build/styles.js

```diff
@@ -7,11 +7,11 @@ const sourcemaps = require('gulp-sourcemaps');
 const rtlcss = require('postcss-rtlcss');
 const postcss = require('gulp-postcss');
 const pump = pify(require('pump'));
+const sass = require('sass-embedded');
```

Comment:
`sass-embedded` is maintained by the same team as `sass`, but is a little bit faster in our workload.